Lesson_2_JSON_YAML_XML/Chinchilla.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: "Simon"
+age: "14.5"
+color: "Gray"
+isIndoor: "true"
+MainFoods:
+- "Hay"
+- "Versele-Laga Complete Chinchilla & Degu"
+- "OXBOW Animal Health Critical Care"
+    
+AdditionalFoods:
+  - "Vitakraft emotion beauty"
+FavoriteFoods:
+- "Carrot" 
+- "Dandelion" 
+- "Apple"
+  
+health: 
+  vaccinated: "false"
+  weightKg: "0.45"
+  medicalCheckup: "true"
+    
```

Comment:
empty lines can be removed here as well to improve readability and keep everything tight enough as it's a one general object